doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -4,4 +4,803 @@ title: Notifications
 permalink: /:path/
 sort_idx: 70
 ---
-Purpose of notifications. Schemas and versioning. Topics, visibility (EP groups related), TTL, and subscriptions (mandatory and optional).
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+
+The Kaa Notification subsystem enables delivery of messages from the Kaa cluster to endpoints (EP). Unlike configuration data that represents the desired EP state, notifications can be thought of as calls for a dynamic EP response. 
```

Comment:
i suggest to replace "response." with "action.". This is not an RPC call. This is just a notification.